frontend/styles/partials/_variables.scss

```diff
@@ -26,6 +27,7 @@ $button-active: #ff9f00;
 $main-blue: #4FB6AC;
 $map-grey: #F1F3F5;
 
+
 // Sizes
```

Comment:
under this section seems like there are : 
$text-dark-color: #4e4e4e;
$text-light-color: $color-white;

Don't think we need these there so not sure why I added them there :( So either copy them up to the // Colors section please? And then I will have a look and see if we need them